### ISSUE TITLE
definition of region in example 42 is probably not correct

### ISSUE THREAD

**Issue by pthopesch** [w3c/ttml-webvtt-mapping#24]
In example 42 the region is defined as follows:
Region: id=regionExample1 width=50% lines=3 regionanchor=0%,0% viewportanchor=0%,0% scroll=up

The correct definition sould be:
REGION
id:regionExample1 width.50% lines:3 regionanchor:0%,0% viewportanchor:0%,0% scroll:up

--> https://w3c.github.io/webvtt/#webvtt-region-definition-block
--> https://w3c.github.io/webvtt/#region-settings

**Comment by andreastai** [comment]
Yes, the mapping does not match the latest draft of the WebVTT spec. The explanation is that the mapping document was based on a previous version of WebVTT where the notation with "=" was correct. 

As it is not clear if the new substantial edit is already implemented somewhere and also if it will make in the final spec version I propose to leave it that way for now but come back to it once the WebVTT spec is stable.
